parlai/agents/repeat_label/repeat_label.py

```diff
@@ -52,9 +51,4 @@ def act(self):
         else:
             reply['text'] = "I don't know."
 
-        if 'label_candidates' in obs and len(obs['label_candidates']) > 0:
```

Comment:
why not do this any more?   it seemed to make sense to me, and repeat_label is only a debugging thing anyway? is there a speed problem or what?